### PR TITLE
#8012: 3D Tiles documentation

### PR DESCRIPTION
**Pull request by allyoucanmap**
## Description

Add documentation and options related to 3d tiles format and fix the navigation tools configuration

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Other... Please describe: dev guide

## Issue

**What is the current behavior?**

#8012

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [x] No

### CODE REVIEW

File: docs/developer-guide/maps-configuration.md [hunk 1]
```diff
@@ -975,6 +976,34 @@ i.e.
 }
 ```
 
+#### 3D tiles
+
+This type of layer shows 3d tiles version 1.0 inside the Cesium viewer. This layer will not be visible inside 2d map viewer types: openlayer or leaflet.
+See specification for more info about 3d tiles: https://www.ogc.org/standards/3DTiles
```
Comment (offtherailz) [inline docs/developer-guide/maps-configuration.md:982]:
```suggestion
See specification for more info about 3d tiles [here](https://www.ogc.org/standards/3DTiles).
```

File: docs/developer-guide/maps-configuration.md [hunk 2]
```diff
@@ -975,6 +976,34 @@ i.e.
...
+      "body": { // 3d tiles style
+        "color": "color('#43a2ca', 1)"
+      }
+    }
+}
+```
+
+The style body object for the format 3dtiles accepts rules described in the 3d tiles styling specification version 1.0 available here:
+
+- https://github.com/CesiumGS/3d-tiles/tree/1.0/specification/Styling
```
Comment (offtherailz) [inline docs/developer-guide/maps-configuration.md:1005]:
```suggestion
The style body object for the format 3dtiles accepts rules described in the 3d tiles styling specification version 1.0 available [here](https://github.com/CesiumGS/3d-tiles/tree/1.0/specification/Styling).
```

**Review by offtherailz — APPROVED** [review]

**Comment by offtherailz** [comment]
LGTM. Use hyperlinks whenever is possible, nobody wants to copy and paste the URL.

**Comment by tdipisa** [comment]
@ElenaGallo please check in dev and let us know for the backport.

**Pull request merged**